src/main/java/com/commerce/song/service/impl/AccountServiceImpl.java

```diff
@@ -91,12 +92,19 @@ public AccountDto.CheckPhoneDupRes checkPhoneDup(String phone) {
     }
 
     @Override
-    public AccountDto.FindEmailRes findEmail() {
-        return null;
+    public AccountDto.FindEmailRes findEmail(AccountDto.FindEmailReq reqDto) {
```

Comment:
try catch로 감싸서 예외처리보다는 
```suggestion
    public AccountDto.FindEmailRes findEmail(AccountDto.FindEmailReq reqDto) {
        Account account =  accountRepository.findByUsernameAndPhone(reqDto.getUsername(), reqDto.getPhone());
        if(account == null) {
          throw new EntityNotFoundException("유저 정보가 없습니다.");
        }
        return new AccountDto.FindEmailRes(account);
    }
```
와 같은 형태가 더 좋지 않을까 생각이 듭니다